lua/entities/gmod_wire_gpu/cl_init.lua

```diff
@@ -440,6 +440,9 @@ local function GPU_DrawHUD()
       local screenWidth = ScrW()
       local screenHeight = ScrH()
 
+      if math.min(1/30,CurTime() - (videoGPU.PreviousTime or 0)) > 0 then 
```

Comment:
This equation looks wrong. Also why not just draw every frame instead of framerate limiting it?

Reply:
Yeah I think it's safe to remove the framelimit check, I was paranoid about making sure I'm calling ent:draw when it will guarantee a rendered frame but hudpaint seems like a safe context(delta-wise) to call the gpu's draw function without a framelimit check.

So, my reasoning was that with the GPU's draw func, it checks the equation(that is present on this line) it doesn't check between last rendered frame and now, only between last draw call and now, so if this gets called quickly enough that it results in a deltatime of less than or equal to 0 it will stall GPU rendering entirely(until called less frequently) rather than stall until time between lastframe and now is within frame limit, this should be reworked soon.